benchmark/benchmark.go

```diff
@@ -0,0 +1,71 @@
+package benchmark
```

Comment:
Can we call this something more descriptive than just benchmark?
Maybe grpcApiBenchmark or something similar.